docs/dev_guide/pr_review_procedure.rst

```diff
@@ -26,6 +26,30 @@ Before the ‘merge’ button is clicked the following criteria must be met:
 
 It is important that approval for merging the PR is done on the comment thread, as this becomes part of the ‘permanent record’, this includes in during community meetings or in chat.
 
+Continuous Integration
+======================
+
+Currently we have a variety of bots or services that respond or activate on an opened pull request.
+While we try not to change them, they have undergone several changes with the aim of making them clear and focused on specific issues.
+
+*  pep8speaks: Performs a PEP8 check on any submitted code.
+*  CircleCi: Tests to see if sunpy installs and builds the documentation.
+*  Giles: Returns a link if the documentation does build successfully.
+*  Travis: Runs our test suite to make sure it passes on Linux and mac OS.
+*  AppVeyor: Runs our test suite to make sure it passes on Windows.
+*  CodeCov: Checks how many lines of the code lack test coverage.
```

Comment:
Should we add links to each of them? It could be either to their website, their documentation or the SunPy page if exists (e.g., codecov `https://codecov.io/gh/sunpy/sunpy/`)